manifests/basic_ucp/ucp-armada.yaml

```diff
@@ -0,0 +1,505 @@
+  release: maas
+  namespace: maas
+  install:
+    no_hooks: false
+  upgrade:
+    no_hooks: false
+  pre:
+    delete: []
+    create: []
+  post:
+    delete: []
+    create: []
+  values:
+    bootdata_url: http://172.20.0.12:31000/api/v1.0/bootdata/
+    credentials:
+      secret:
+        namespace: drydock
+    images:
+      maas_rack: quay.io/attcomdev/maas-rack:2.2.0-no-ipmi-patch
+    labels:
+      node_selector_key: ucp-control-plane
+      node_selector_value: enabled
+    network:
+      proxy:
+        node_port:
+          enabled: true
+          port: 31800
+      gui:
+        node_port:
+          enabled: true
+          port: 31900
+    conf:
+      maas:
+        url: http://172.20.0.12:31900/MAAS
+        proxy:
+          enabled: 'false'
+          server: http://one.proxy.att.com:8080
+        ntp:
+          servers: ntp.ubuntu.com
+        dns:
+          upstream_servers: 8.8.8.8
+      secret: 3858a12230ac3c915f300c664f12063f
+  source:
+    type: local
+    location: /opt/armada/charts
+    subpath: maas
+  dependencies:
+    - helm-toolkit
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: drydock
+data:
+  chart_name: drydock
+  release: drydock
+  namespace: ucp
+  install:
+    no_hooks: false
+  upgrade:
+    no_hooks: false
+  pre:
+    delete: []
+    create: []
+  post:
+    delete: []
+    create: []
+  values:
+    images:
+      drydock: quay.io/attcomdev/drydock:feature_DRYD19_keystone_middleware_integration
+    labels:
+      node_selector_key: ucp-control-plane
+      node_selector_value: enabled
+    network:
+      drydock:
+        node_port:
+          enabled: true
+          port: 31000
+    conf:
+      drydock:
+        maasdriver:
+          drydock_provisioner:
+            maas_api_url: http://172.20.0.12:31900/MAAS/api/2.0/
+  source:
+    type: local
+    location: /opt/armada/charts
+    subpath: drydock
+  dependencies:
+    - helm-toolkit
+---
+schema: armada/Manifest/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-basic
+data:
+  release_prefix: armada-ucp
+  chart_groups:
+    - ceph
+    - ceph-bootstrap
+    - ucp-infra
+    - ucp-services
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ceph
+data:
+  description: 'Storage Backend'
+  chart_group:
+    - ceph
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ceph-bootstrap
+data:
+  description: 'Storage Backend Config'
+  chart_group:
+    - ucp-ceph-config
+    - maas-ceph-config
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-infra
+data:
+  description: 'UCP Infrastructure'
+  chart_group:
+    - ucp-mariadb
+    - ucp-memcached
+    - maas-postgresql
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-services
+data:
+  description: 'UCP Services'
+  chart_group:
+    - maas
+    - drydock
+    - ucp-keystone
+...
```

Comment:
I prefer explicit syntax vs implicit syntax. The `...` is an explicit YAML end-document token.